test/e2e/image_pull_error_test.go

```diff
@@ -61,7 +62,8 @@ func TestImagePullError(t *testing.T) {
 		if cond != nil && !cond.IsUnknown() {
 			if cond.IsFalse() {
 				if strings.Contains(cond.Message, backoffMsg) ||
-					strings.Contains(cond.Message, daemonMsg) {
+					strings.Contains(cond.Message, errMsg1) ||
+					strings.Contains(cond.Message, errMsg2) {
```

Comment:
Should we ditch the specific error message check altogether? It's going to be relying on the underlying CRI and not be deterministic in that sense. I feel like verifying we're in a "Back-off pulling image" situation is enough for this test. The specific error is not necessarily important imo.